buster/documents_manager/base.py

```diff
@@ -28,13 +29,17 @@ def get_embedding_openai(text: str):
 
 @dataclass
 class DocumentsManager(ABC):
+    def __init__(self, required_columns=list[str], csv_checkpoint: Optional[str] = None):
```

Comment:
did you mean to use `:` instead of `=` for required_columns?

Reply:
yes fixing